types/js-cookie/index.d.ts

```diff
@@ -32,6 +33,18 @@ declare namespace Cookies {
          * secure protocol (https). Defaults to false.
          */
         secure?: boolean;
+
+        /**
+         * Define the first party cookie policy.
+         * @see: https://tools.ietf.org/html/draft-west-first-party-cookies-07
+         */
+        samesite?: 'Strict' | 'Lax';
```

Comment:
Consider naming the property `sameSite`, since in the spec it is two words (the official attribute name is "SameSite").

Reply:
The property should support a value of "None" in addition to Strict and Lax.

Reply:
Agreed for the property name, I asked myself the same question and went for lowercase because `secure` was used instead of `Secure`. CamelCase would make more sense though.

For `None` you're right, I was looking at an outdated RFC's BNF (https://tools.ietf.org/html/draft-west-first-party-cookies-07#section-3.1)